gsplat/rasterize.py

```diff
@@ -117,10 +115,7 @@ def forward(
         num_intersects, cum_tiles_hit = compute_cumulative_intersects(num_tiles_hit)
 
         if num_intersects < 1:
-            out_img = (
-                torch.ones(img_height, img_width, colors.shape[-1], device=xys.device)
-                * background
-            )
+            out_img = torch.ones(img_height, img_width, colors.shape[-1], device=xys.device) * background
```

Comment:
Should we change the annotation of 
`background: Optional[Float[Tensor, "channels"]] = None` to `background: Float[Tensor, "channels"]`. The implementation seems will rather if background is None.